lib/locales/ru.js

```diff
@@ -0,0 +1,194 @@
+
+i18n.translations.ru = {
```

Comment:
I'm sorry, i don't know about it. So, this PR becomes bad thing, please decline in and i'll make one another.